src/components/SidePanel.js

```diff
@@ -1,24 +1,25 @@
-import React, { useState, useContext } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import {
   Box,
+  Button,
```

Comment:
I don't mind you committing linter changes like this if there are functional changes to a file, but in this case it's just the ordering of `import` statements that has changed. That tends to cause merge headaches for other developers.